unimol/unimol/losses/docking_pose.py

```diff
@@ -37,8 +38,9 @@ def forward(self, model, sample, reduce=True):
 
         ### holo distance loss
         holo_distance_mask = sample["target"]["holo_distance_target"].ne(
-            0
```

Comment:
Add these two lines to the `__init__` of the loss
```
self.eos_idx = task.dictionary.eos()
self.bos_idx = task.dictionary.bos()
```
And use `token_mask` to define `holo_distance_mask` may be better. bc such modification does not need to define other `pad idx` that might cause confusion. and does not need to change the `task` file
```
token_mask = sample["net_input"]["src_tokens"].ne(self.padding_idx) & sample["net_input"]["src_tokens"].ne(self.eos_idx) & sample["net_input"]["src_tokens"].ne(self.bos_idx)
sz = token_mask.size()
holo_distance_mask = torch.zeros(sz[0], sz[1], sz[1]).type_as(token_mask)
holo_distance_mask = token_mask.unsqueeze(-1) & token_mask.unsqueeze(1)
```

Reply:
Yes, thanks for the tip. 
I have modified this part.